tlm_teamd/teamdctl_mgr.cpp

```diff
@@ -5,6 +5,12 @@
 
 #include "teamdctl_mgr.h"
 
+#define MAX_RETRY 3
+
+/// Store the last errored lag name and the retry count.
+static std::string last_errored_lag_name= std::string("");
+static int no_of_retry = 0;
```

Comment:
wonder if we can move all these static variable into get_dump() function?

Reply:
We would need to access this variables in remove_lag API also.

Reply:
I have kept these variables global ( it is not static anymore, in updated code.). It is used in remove_lag() and get_dump() APIs